modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/sequence/steps/StepConfigTable.scala

```diff
@@ -79,54 +78,47 @@ object StepConfigTable {
   }
 
   val TableColumnMeta: ColumnMeta[TableColumn] = ColumnMeta[TableColumn](
-    NameColumn,
-    name = "name",
-    label = "Name",
+    column  = NameColumn,
+    name    = "name",
+    label   = "Name",
     visible = true,
-    PercentageColumnWidth.Half)
+    width   = PercentageColumnWidth.Half)
 
   val ValueColumnMeta: ColumnMeta[TableColumn] = ColumnMeta[TableColumn](
-    ValueColumn,
-    name = "value",
-    label = "Value",
+    column  = ValueColumn,
+    name    = "value",
+    label   = "Value",
     visible = true,
-    PercentageColumnWidth.Half)
+    width   = PercentageColumnWidth.Half)
 
   val InitialTableState: TableState[TableColumn] = TableState[TableColumn](
-    NotModified,
-    0,
-    NonEmptyList.of(TableColumnMeta, ValueColumnMeta))
-
-  private def columns(b: Backend): List[Table.ColumnArg] = {
-    val width = b.props.size.width
-    // Tell the model to resize a column
-    def resizeRow(c: TableColumn): (String, JsNumber) => Callback =
-      (_, dx) => {
-        val percentDelta = dx.toDouble / width
-        val s            = b.state.applyOffset(c, percentDelta)
-        b.setState(s) >> SeqexecCircuit.dispatchCB(
-          UpdateStepsConfigTableState(s))
-      }
+    userModified   = NotModified,
+    scrollPosition = 0,
+    columns        = NonEmptyList.of(TableColumnMeta, ValueColumnMeta))
+
+  private def colBuilder(b: Backend): ColumnRenderArgs[TableColumn] => Table.ColumnArg = tb => {
+    val state = b.state
+    def updateState(s: TableState[TableColumn]): Callback =
+      b.setState(s) >> SeqexecCircuit.dispatchCB(UpdateStepsConfigTableState(s))
```

Comment:
If you're using `>>` because you need the RHS to be by-name then it's probably worth a comment. Otherwise prefer `*>`.